src/components/PortfolioAccordions.vue

```diff
@@ -122,14 +129,15 @@
   import * as d3Base from 'd3';
```

Comment:
same D3 comment

Reply:
Here `d3` is used to read in the csv. We could add a workaround w/ `jQuery` or an npm module, but I think I'll leave as is for now, since we're planning to move away from the csv approach, and can drop `d3` then